src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/target.py

```diff
@@ -121,7 +129,7 @@ def _target_is_null_path(self):
         self.path = '/deployment' # TODO we will fetch this from parameter store
         responses = self.organizations.dir_to_ou(self.path)
         self._create_response_object(responses)
-
+    #pylint: disable=R0911
     def fetch_accounts_for_target(self):
```

Comment:
Could you add your new use case in the tests found at `src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/tests/test_target.py`?